iothub/service/src/Feedback/Models/FeedbackRecord.cs

```diff
@@ -8,67 +8,53 @@ namespace Microsoft.Azure.Devices
 {
     /// <summary>
     /// Contains properties of an individual feedback record.
-    /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
     /// </summary>
+    /// <remarks>
+    /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
+    /// </remarks>
     public class FeedbackRecord
     {
         /// <summary>
         /// The Id of the cloud-to-device message to which this feedback message relates.
-        /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
         /// </summary>
+        /// <remarks>
+        /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
+        /// </remarks>
         [JsonProperty(PropertyName = "originalMessageId", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public string OriginalMessageId { get; set; }
+        public string OriginalMessageId { get; internal set; }
```

Comment:
While we still need to figure out what our mocking story is for data models, we'll make these setters internal for now.